src/org/opensolaris/opengrok/web/PageConfig.java

```diff
@@ -1277,12 +1290,12 @@ public File getDataRoot() {
      *
      * @return a search helper.
      */
-    public SearchHelper prepareSearch() {
+    public SearchHelper prepareSearch(boolean isSearchAll) {
```

Comment:
I would rather get this boolean value inside this function (request is a member variable in PageConfig) and keep this method without the parameter